static/app/views/settings/projectAlerts/settings.tsx

```diff
@@ -1,168 +1,176 @@
-            </Button>
-          }
-        />
-        <PermissionAlert project={project} />
-        <AlertLink to="/settings/account/notifications/" icon={<IconMail />}>
-          {t(
-            'Looking to fine-tune your personal notification preferences? Visit your Account Settings'
-          )}
-        </AlertLink>
-
-        <Form
-          saveOnBlur
-          allowUndo
-          initialData={{
-            subjectTemplate: project.subjectTemplate,
-            digestsMinDelay: project.digestsMinDelay,
-            digestsMaxDelay: project.digestsMaxDelay,
-          }}
-          apiMethod="PUT"
-          apiEndpoint={projectEndpoint}
-        >
-          <JsonForm
-            disabled={!canEditRule}
-            title={t('Email Settings')}
-            fields={[fields.subjectTemplate]}
-          />
-
-          <JsonForm
-            title={t('Digests')}
-            disabled={!canEditRule}
-            fields={[fields.digestsMinDelay, fields.digestsMaxDelay]}
-            renderHeader={() => (
-              <PanelAlert type="info">
-                {t(
-                  'Sentry will automatically digest alerts sent by some services to avoid flooding your inbox with individual issue notifications. To control how frequently notifications are delivered, use the sliders below.'
-                )}
-              </PanelAlert>
-            )}
-          />
-        </Form>
+  const handleDisablePlugin = (plugin: Plugin) => {
+    updatePlugin(plugin, false);
+  };
 
-        {canEditRule && (
-          <PluginList
-            organization={organization}
-            project={project}
-            pluginList={(pluginList ?? []).filter(
-              p => p.type === 'notification' && p.hasConfiguration
-            )}
-            onEnablePlugin={this.handleEnablePlugin}
-            onDisablePlugin={this.handleDisablePlugin}
-          />
+  return (
+    <Fragment>
+      <SentryDocumentTitle
+        title={routeTitleGen(t('Alerts Settings'), projectSlug, false)}
+      />
+      <SettingsPageHeader
+        title={t('Alerts Settings')}
+        action={
+          <LinkButton
+            to={{
+              pathname: `/organizations/${organization.slug}/alerts/rules/`,
+              query: {project: project?.id},
+            }}
+            size="sm"
+          >
+            {t('View Alert Rules')}
+          </LinkButton>
+        }
+      />
+      <PermissionAlert project={project} />
+      <AlertLink to="/settings/account/notifications/" icon={<IconMail />}>
+        {t(
+          'Looking to fine-tune your personal notification preferences? Visit your Account Settings'
         )}
-      </Fragment>
-    );
-  }
+      </AlertLink>
+
+      {isProjectLoading || isPluginListLoading ? (
+        <LoadingIndicator />
+      ) : (
+        <Fragment>
+          <Form
+            saveOnBlur
+            allowUndo
+            initialData={{
+              subjectTemplate: project.subjectTemplate,
```

Comment:
Does the project cache get updated after these changes are made? I'm wondering what happens if you modify a setting here, go to another page, then come back. The form might see the old state and not update when the project refetch occurs.

Reply:
set cacheTime to 0